python/pyspark/sql/catalog.py

```diff
@@ -791,7 +791,7 @@ def createTable(
         schema: Optional[StructType] = None,
         description: Optional[str] = None,
         **options: str,
-    ) -> DataFrame:
+    ) -> "DataFrame":
```

Comment:
why do we need to change this?

Reply:
Otherwise the `inspect.getmembers` returns different object as below:
```
pyspark.sql.dataframe.DataFrame
pyspark.sql.connect.dataframe.DataFrame
```